Ladybird/Qt/Tab.cpp

```diff
@@ -115,32 +115,7 @@ Tab::Tab(BrowserWindow* window, WebContentOptions const& web_content_options, St
         m_hover_label->hide();
     };
 
-    view().on_load_start = [this](const URL::URL& url, bool is_redirect) {
```

Comment:
This is removing more than just history handling - for example, setting the title and favicon.

Reply:
that is intentional change. now we won't immediately update favicon and title but do that once document is loaded in [HTMLTitleElement::children_changed()](https://github.com/SerenityOS/serenity/blob/master/Userland/Libraries/LibWeb/HTML/HTMLTitleElement.cpp#L29C6-L29C40) and favicon is updated from [Document::update_readiness()](https://github.com/SerenityOS/serenity/blob/master/Userland/Libraries/LibWeb/DOM/Document.cpp#L2171)

Reply:
That at least breaks navigating from a page with a favicon to a page without a favicon. For example, navigate to xkcd.com then example.com - the favicon for xkcd will remain on the tab.

Reply:
right, we still have to reset favicon and title after navigation. thank you for catching this.